File: truckdoc-client-m2/truckdoc-client-updater/src/main/java/com/sanda/truckdoc/client/updater/network/UpdateResponse.java
package com.sanda.truckdoc.client.updater.network;

import com.google.gson.annotations.SerializedName;

/**
 * Created by astra on 05.07.2015.
 */
public class UpdateResponse {

    @SerializedName("updateAvailable")
    public boolean updateAvailable;

    @SerializedName("updateUrl")
    public String url;

    @SerializedName("updateVersionCode")
    public int versionCode;
}

File: truckdoc-client-m2/truckdoc-client-updater/src/main/java/com/sanda/truckdoc/client/updater/network/UpdateResponse.kt
package com.sanda.truckdoc.client.updater.network

import com.google.gson.annotations.SerializedName

/**
 * Created by astra on 05.07.2015.
 */
data class UpdateResponse(
        @SerializedName("updateAvailable")
        @JvmField
        val updateAvailable: Boolean = false,
        @JvmField
        @SerializedName("updateUrl")
        val url: String? = null,
        @JvmField
        @SerializedName("updateVersionCode")
        val versionCode: Int = 0
)
